test/kokoro/xds_k8s_install_test_driver.sh

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env bash
+# Copyright 2020 gRPC authors.
```

Comment:
- Were these two shell scripts copied from somewhere?
- Do we _absolutely need_ to maintain separate copies here?  Why?
- If so, should they be reviewed and updated here?

Reply:
You're absolutely right. I have a ticket to take care of that, internal ref b/186553728.